torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -19,6 +19,8 @@ namespace rpc {
 
 constexpr long kToMilliseconds = 1000;
 
+const std::string kGilAverageWaitTime = "agent.gil_average_wait_time_us";
```

Comment:
nit: maybe all use `const` or `constexpr` to be consisent?

Reply:
Looking elsewhere in the RPC code, it seems like we use `const` for strings and `constexpr` for numerical values?

Reply:
I don't think strings can be constexprs, as they have a non-constexpr constructor/destructor (because they are heap allocated). But please correct me if I'm wrong because I'm not so familiar with constexpr...